docs/tutorials/basic-dapp/part-4-use-external-accounts.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Part 4 - Use external accounts"
+---
+
+In this tutorial, we do no show how to create accounts using this `api` although it's possible. The reason is simple, we do not want to encourage developers to manage accounts in the browser. DNS attacks, pishing and other common attacks put users at risk. If the use case of the DApp allows it, we suggest using injected accounts instead. Users will create and manage their accounts externally to the DApp, we will simply use these accounts if the user accepts it. In this Part, we will show how you can show external accounts balance and send a transaction.
+
+## 4.1 Get Polkadot-js extension
+
+Get into our user's shoes and install the [Polkadot-js enxtension](https://github.com/polkadot-js/extension) from the [Chrome](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd?hl=en) or [Firefox](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/) store. 
+Create an account using the extension and give it a name.
+
+We will now adapt our application to inject any account created externally in our current interface.
+
+This is how the extension looks like with 2 accounts creatively named Bob and Alice:
+![Pokadot-js extension](/docs/tutorials/basic-dapp/assets/part-3-1.jpg)
+
+## 4.2 Display external account's balances
+
+Externally generated accounts loaded thank to [`@polkadot/extension-dapp`](https://github.com/polkadot-js/extension/tree/master/packages/extension-dapp).
+```bash
+yarn add @polkadot/extension-dapp@beta
+```
+
+This package gives us access to `web3Enable` to declare our DApp and request the right to read the accounts from our users. `isWeb3Injected` lets us know if our users have an extension supporting the `@polkadot/extension-dapp` protocol, and finally the `web3Accounts` function that should be called if the user has a compatible extension, and has granted us access to their accounts.
+
+To keep things simple, we will not display anything else than a loader to our users while the authorization is requested. This is a blocking experience that should not be reproduced in a real DApp. Rather show something to your users, and patiently wait for them to accept the extension request.
+
+We will introduce a new state variable `accountLoaded` to show this loader in case our user have an extension but haven't granted us access to their accounts.
```

Comment:
```suggestion
We will introduce a new state variable `accountLoaded` to show this loader in case our user has an extension but hasn't granted us access to their accounts.
```